modules/core/src/main/java/org/apache/ignite/cache/query/IndexQuery.java

```diff
@@ -30,12 +30,16 @@
 
 /**
  * Index queries work over distributed indexes and retrieve cache entries that match the specified criteria.
- * {@code QueryCursor} delivers sorted cache entries by the order defined for queried index.
+ * <p>{@code QueryCursor} delivers sorted cache entries by the order defined for queried index.
  *
- * {@code IndexQuery} has to be initialized with cache value class or type. The algorithm of discovering index is as follows:
- * 1. If {@link #idxName} is set, then use it.
- * 2. If {@link #idxName} is not set, then find an index that matches criteria fields.
- * 3. If neither {@link #idxName}, nor {@link #setCriteria(List)} is used, then perform index scan over PK index for specified Value type.
+ * <p>{@code IndexQuery} has to be initialized with cache value class or type. The algorithm of discovering index is
```

Comment:
Here and above, please check how this tag is used in other classes. For example, you can check classes `ContinuousQueryWithTransformer` or `QueryCursor`.

I mean this tag usually occupies whole line, no symbols are present on the same line.

Reply:
@timoninmaxim The changes have been made, each <p> tag occupies a whole line now.